project/Dependencies.scala

```diff
@@ -132,7 +132,7 @@ object Dependencies {
   val billingProfileManager = excludeJakarta("bio.terra" % "billing-profile-manager-client" % "0.1.175-SNAPSHOT")
   val terraCommonLib = tclExclusions(excludeJakarta("bio.terra" % "terra-common-lib" % "0.0.93-SNAPSHOT" classifier "plain"))
   val sam: ModuleID = excludeJakarta("org.broadinstitute.dsde.workbench" %% "sam-client" % "0.1-d606036")
-  val leonardo: ModuleID = "org.broadinstitute.dsde.workbench" % "leonardo-client_2.13" % "1.3.6-8740f99-SNAP"
+  val leonardo: ModuleID = "org.broadinstitute.dsde.workbench" % "leonardo-client_2.13" % "1.3.6-7579402"
```

Comment:
@Qi77Qi why did you update this to a SNAP version? Any concerns about going to this version?

Reply:
the snap is from https://github.com/DataBiosphere/leonardo/pull/3565

We need to get WDS, cromwell etc to use the snap client from that PR first and upgrade all existing apps to use that snap client before we can merge that PR

Reply:
OK, I will close this PR. But once you are are done with your work, can you update Rawls to a non-snap version so it is clear to the Rawls maintenance team that your work is complete?

Reply:
Will do!